CAHNGELOG.md

```diff
@@ -0,0 +1,689 @@
+# v0.7.0-alpha
```

Comment:
`CAHNGELOG.md` ?

Reply:
Will need to amend the PR now that we have branched off `0.7`.